lib/active_model/serializer/adapter.rb

```diff
@@ -63,7 +63,7 @@ def is_fragment_cached?
       end
 
       def cache_key
-        (@klass._cache_key) ? "#{@klass._cache_key}/#{@cached_serializer.object.id}-#{@cached_serializer.object.updated_at}" : @cached_serializer.object.cache_key
+        (@klass._cache_key) ? "#{@klass._cache_key}/#{@cached_serializer.object.id}-#{@cached_serializer.object.updated_at.strftime("%Y%m%d%H%M%S%9N")}" : @cached_serializer.object.cache_key
```

Comment:
Unfortunately this means if `updated_at` is `nil`, you're gonna have a Bad Time. If that's something we should consider, I can update this method to be a bit more resilient.